vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/StaffSearchModule.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.vspace.core.model.impl;
```

Comment:
This class does not belong into the model package. The model package contains the domain classes (the different elements of an exhibition). This class is mainly for transfer results from the search.